src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -106,6 +107,19 @@ export class HubConnectionBuilder {
         return this;
     }
 
+     /** Configures the {@link @aspnet/signalr.HubConnection} to automatically attempt to reconnect if the connection is lost. */
+    public withAutomaticReconnect(): HubConnectionBuilder {
```

Comment:
How common do you think people would write the following code? And accidentally overwrite their reconnect policy?
```typescript
builder.withUrl("/chat", { reconnectPolicy: new CustomReconnect() })
    .withAutomaticReconnect()
    .build();
```

Reply:
Not very common, but I can see how it could happen. Do you think we should have withAutomaticReconnect no-op if a reconnectPolicy is already set?